dev-db/mydumper/mydumper-0.10.7.2.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P="${PN}-$(ver_rs 3 -)"
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
```

Seeing the later clause in mydumper.h
```suggestion
LICENSE="GPL-3+"
```

---

Good catch, thanks.